<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Jagdamba" failures="1" tests="11" name="loginPom.TestNinja" time="3.393" errors="0" timestamp="2023-03-07T23:30:38 IST" skipped="2">
  <testcase classname="loginPom.TestNinja" name="passverify" time="0.253"/>
  <system-out/>
  <testcase classname="loginPom.TestNinja" name="verifytelephone" time="0.199"/>
  <system-out/>
  <testcase classname="loginPom.TestNinja" name="enterfname" time="0.416"/>
  <system-out/>
  <testcase classname="loginPom.TestNinja" name="firstname" time="0.288"/>
  <system-out/>
  <testcase classname="loginPom.TestNinja" name="emailbox" time="0.313"/>
  <system-out/>
  <testcase classname="loginPom.TestNinja" name="checkboxno" time="0.001">
    <skipped/>
  </testcase> <!-- checkboxno -->
  <system-out/>
  <testcase classname="loginPom.TestNinja" name="policybox" time="0.002">
    <skipped/>
  </testcase> <!-- policybox -->
  <system-out/>
  <testcase classname="loginPom.TestNinja" name="confpass" time="0.224"/>
  <system-out/>
  <testcase classname="loginPom.TestNinja" name="checkboxyes" time="0.061">
    <failure message="The following asserts failed:
	expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at loginPom.TestNinja.checkboxyes(TestNinja.java:107)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkboxyes -->
  <system-out/>
  <testcase classname="loginPom.TestNinja" name="checkmyaccount" time="0.794"/>
  <system-out/>
  <testcase classname="loginPom.TestNinja" name="register" time="0.842"/>
  <system-out/>
</testsuite> <!-- loginPom.TestNinja -->
